bitrise.yml

```diff
@@ -105,11 +105,8 @@ workflows:
 
   # ----------------------------------------------------------------
   # --- Utility workflows
-
```

Comment:
Why did this change?

Reply:
Just a cosmetic fix, other workflows with comment did not have the extra line, for example line 127

Reply:
I was referring to the removed description :) but ok